apollo-router/src/router/event/schema.rs

```diff
@@ -154,6 +159,25 @@ impl SchemaSource {
                 .filter_map(|s| async move { s.map(Event::UpdateSchema) })
                 .boxed()
             }
+            SchemaSource::OCI(oci_config) => {
+                futures::stream::once(async move {
+                    // TODO: Fix error handling
```

Comment:
Interestingly, the other mechanisms seem to always return `None` when a schema can't be retrieved. Without digging into that too much, I'm assuming that is handled elsewhere, so your error handling might already be "correct".

We can dig into that when you get a bit further along.